src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/README.md

```diff
@@ -53,20 +54,3 @@ const onChange = (value) => setState({selectValue: value});
     <Option value="22">Godric Gryffindor</Option>
 </Select>
 ```
-
-Because the select doesn't change its label itself, its fairly straight forward to provide a select
```

Comment:
When just dealing with single selects, no special components have to be created to create actions (clickable elements which do not change the value of the select). However when also having multi selects, which need actions, this is not possible anymore. Therefore I created a designated `Action` component.